project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "3.0.0-beta2-SNAPSHOT"
+(defproject clojurewerkz/elastisch "5.0.0-alpha1-SNAPSHOT"
```

Comment:
*Never* alter version numbers in a pull request. It's up to the maintainer to decide when to do it and what the next version should be.